recipes/blend2d/all/CMakeLists.txt

```diff
@@ -4,7 +4,4 @@ project(cmake_wrapper LANGUAGES CXX)
 set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
 set(CMAKE_CXX_STANDARD 11)
 
-include(conanbuildinfo.cmake)
-conan_basic_setup(KEEP_RPATHS)
-
-add_subdirectory(source_subfolder)
+add_subdirectory(src)
```

Comment:
please remove this file and call CMake configuration on upstream CMakeLists directly.

You can inject CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS & CMAKE_CXX_STANDARD in the toolchain.